docs/advanced-features/custom-error-page.md

```diff
@@ -25,7 +25,7 @@ Error.getInitialProps = ({ res, err }) => {
 export default Error
 ```
 
-> `pages/_error.js` is only used in production. In development you'll get an error with the call stack to know where the error originated from.
+> `pages/_error.js` is only used in production for non-404 errors. In development you'll get an error with the call stack to know where the error originated from.
```

Comment:
In the beginning, it says this:

> **404** or **500** errors are handled both client-side and server-side by the `Error` component. If you wish to override it, define the file `pages/_error.js` and add the following code:

Should we remove 404 from the above sentence?

Reply:
Yeah seems like it should be removed now 👍